src/components/static/pages/Faq.tsx

```diff
@@ -28,50 +29,43 @@ export class Faq extends React.Component {
                 </div>
               </ContentWrapper>
               <ContentWrapper>
-                <div className="question">
-                  What does Nuevo Foundation offer?
-                </div>
+                <div className="question">What does Nuevo Foundation offer?</div>
               </ContentWrapper>
               <ContentWrapper>
                 <div className="questiontext">
-                  Nuevo Foundation offers workshops for kids K-12 where they get
-                  to learn about coding and technology.
+                  Nuevo Foundation offers workshops for kids K-12 where they get to learn about coding and technology.
                 </div>
               </ContentWrapper>
               <ContentWrapper>
                 <div className="question">What is a virtual session?</div>
               </ContentWrapper>
               <ContentWrapper>
                 <div className="questiontext">
-                  A virtual session is an opportunity for your students to talk
-                  to someone at Microsoft or any other tech company over
-                  Teams/Skype/Hangouts/Zoom who is excited and passionate about
-                  technology and sharing their experience at their workplace.
-                  You can use this as a way for your students to learn about
-                  what it's like to work in technology. We also use the time to
-                  have the students ask questions about tech or anything they're
-                  curious about.{" "}
+                  A virtual session, or Skype in the Classroom, is an opportunity for your students to talk to someone
+                  at Microsoft over Skype who is excited and passionate about technology and sharing their experience at
+                  Microsoft. You can use this as a way for your students to learn about what it’s like to work in
+                  technology. We also use the time to have the students ask questions about tech or anything they’re
+                  curious about
                 </div>
               </ContentWrapper>
               <ContentWrapper>
                 <div className="question">Who does Nuevo Foundation serve?</div>
               </ContentWrapper>
-              <ContentWrapper>
+              <ContentWrapper className="final-question">
                 <div className="questiontext">
-                  Anyone interested in STEM with a focus on underserved
-                  communities.
+                  Nuevo Foundation was created by a passionate group of friends that are here to serve anyone that is
+                  interested in STEM with a focus on underserved communites.
                 </div>
               </ContentWrapper>
+              <ContentWrapper className="Nuvi">
+                <img id="Nuvi" src={Nuvi} alt="Nuvi" />
+              </ContentWrapper>
```

Comment:
This image shows at the bottom right after all the text. Can we add it on the right side of the text?
![image](https://user-images.githubusercontent.com/6943644/57204325-e70bba00-6f6b-11e9-8830-aa80a7e63572.png)